Formula/fuse-ext2.rb

```diff
@@ -0,0 +1,60 @@
+class FuseExt2 < Formula
+  desc "Multi OS FUSE module to mount ext2, ext3 and ext4 file system devices and/or images with read write support"
+  homepage "https://github.com/alperakcan/fuse-ext2"
+  url "https://github.com/alperakcan/fuse-ext2/archive/e7d031fb34d1c3b8e035bd2edd84ea008d42fae9.tar.gz"
```

Comment:
This needs to be a tagged, stable release not a commit.

Reply:
Ok, I will ask maintainer to tag the project, but it was last done in 2015...